src/script_lua.c

```diff
@@ -1211,6 +1212,12 @@ static int luaSetResp(lua_State *lua) {
     return 0;
 }
 
+/* redis.monotonic() */
+static int redis_monotonic(lua_State *L) {
+    lua_pushnumber(L, (lua_Number)getMonotonicUs());
```

Comment:
This often will expose the real time, which would rather avoid so folks don't try to use it. I would prefer we normalize the behavior so that the first call always returns zero, and subsequent calls always return the relative time from the first call.

Reply:
Done with https://github.com/redis/redis/pull/12971/commits/da70aad07bb8526c34e62546f090d3b6b442ff12

Do you mean using only one lua method for start and end time measure?
I'd prefer to use both methods, to minimize ambiguity.

Reply:
Your proposal solves my concern.

Reply:
Now redis.elapsed_start() doesn't return a lua userdata instead of monotonic clock, the caller can't see the real time.
is that what you mean?

Reply:
The monotonic clock could still be used between lua functions and I was hoping we could also try to nudge people to avoid that by just always starting our monotonic clock at zero.

Reply:
sorry, i can't quite understand what you mean.

```lua
-- timer will be a userdata, the caller can only see its address, e.g. print(timer)
-- There is no way for the caller to use it in any other lua functions, except redis.elapsed_us()
local timer = redis.elapsed_start()
...
do something
...
return redis.elapsed_us(timer)
```

Reply:
The current iteration looks right to me, I think I just misunderstood the update. Thanks.